src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/TileJSON.java

```diff
@@ -129,4 +131,16 @@ public void setBounds(double[] bounds) {
     public void setCenter(double[] center) {
         this.center = center;
     }
+
+    public String getFormat() {
+        return format;
+    }
+
+    public void setFormat(String format) {
+        this.format = format;
+    }
+
+    public String getTilejson() {
```

Comment:
The value we have in the code is "3.0.0" because during the testebed we were looking at a v3 spec that Mapbox never completed and published. Maybe best to go down to 2.2.0, the last official spec: https://github.com/mapbox/tilejson-spec/tree/master/2.2.0

Reply:
fixed